Makefile

```diff
@@ -216,7 +216,7 @@ check:
 
 pytest: $(ALL_PROGRAMS)
 ifndef PYTEST
-	@echo "py.test is required to run the integration tests, please install using 'pip3 install -r tests/requirements.txt'"
+	@echo "py.test is required to run the integration tests, please install using 'pip3 install -r tests/requirements.txt', and rerun 'config'."
```

Comment:
The script is `configure` and not `config`, is it not?

Reply:
:man_facepalming: of course you're right. will fix it.